### PR TITLE
Exclude symlinks

### LINKED ISSUE #6: Exclude symlinks
Say `foo` symlinks to a directory that does not exist before compilation, and my compilation CI is not on GitHub Actions, then checkout followed by conflict finder fails with:
> grep: ./foo: No such file or directory

Is it worth excluding symlinks by default?

### PR DESCRIPTION
**Pull request by andycall**
Looks like there are missing PR about  #https://github.com/olivernybroe/action-conflict-finder/issues/6

Fixed https://github.com/olivernybroe/action-conflict-finder/issues/6

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: 'Merge Conflict Finder'
+name: 'Merge Conflict Finder 2'
```
Comment (olivernybroe) [inline action.yml]: Could you remove the number 2 added here?

**Review by olivernybroe — COMMENTED** [review]
Oh yeah, totally forgot about this, thanks!

File: action.yml (hunk 1 above)
Comment (andycall) [reply action.yml]: Fixed

**Comment by olivernybroe** [comment]
Awesome, need release has been made with this change 👍

**Pull request merged**